### DISCUSSION TITLE
Example of SVG to PNG only bring white screen

### DISCUSSION THREAD

**Discussion by vaporizel** [chillerlan/php-qrcode#247] [Q&A]
im using code from here https://github.com/chillerlan/php-qrcode/blob/v5.0.x/examples/imagickConvertSVGtoPNG.php , its only gives blank screen. if i take out the png header its show like this :

![image](https://github.com/chillerlan/php-qrcode/assets/54837437/52d5e2d9-4751-4c2f-9ed5-a92968ae3bf9)

**Comment by codemasher** [comment]
Hey, did you make any modifications to the example? Are you running it with the intended php-qrcode version? The output is supposed to be the same as the simple SVG example, just converted to PNG:

![imagickConvertSVGtoPNG](https://github.com/chillerlan/php-qrcode/assets/592497/fccfee25-60cc-40bb-9e87-94e134bd5d20)

or the raw console output:
![grafik](https://github.com/chillerlan/php-qrcode/assets/592497/962a0481-dee9-4fcb-b40b-764080437271)

**Comment by vaporizel** [comment]
Hi, im only changed
`require_once __DIR__.'/../vendor/autoload.php';`
to
`require_once __DIR__.'/vendor/autoload.php';`

everything else is as is

*php 8.1 using branch 5.0

**Reply by codemasher** [reply]
Hmm, is ImageMagick properly installed and/or have you tried a different version of it?

**Reply by vaporizel** [reply]
hi, this is installed version :
php8.1-imagick/focal,now 3.7.0-4+ubuntu20.04.1+deb.sury.org+2 amd64

**Reply by codemasher** [reply]
Do the other ImageMagick examples work properly?

**Comment by hosmelq** [comment]
Same error for me.

Running on Mac with:

PHP: 8.3
chillerlan/php-qrcode: 5.0.2

![imagickConvertSVGtoPNG](https://github.com/chillerlan/php-qrcode/assets/1166143/3c57afd2-db1d-4e0d-ba16-59fa9ed65399)

![CleanShot 2024-03-19 at 4  45 06@2x](https://github.com/chillerlan/php-qrcode/assets/1166143/70d8d13d-f012-407f-a783-f38b28ca88bc)

**Reply by hosmelq** [reply]
I tested another Imagick example `http://localhost:3000/imagickWithLogo.php`.

![imagickWithLogo](https://github.com/chillerlan/php-qrcode/assets/1166143/86c0b139-7c42-42bf-bbea-ab404dd30a69)

**Reply by codemasher** [reply]
ImageMagick is not always reliable when converting SVG, see https://github.com/chillerlan/php-qrcode/discussions/216#discussioncomment-7044016
